### PR TITLE
Feat chart view

### LINKED ISSUE #4: Add chart view

### PR DESCRIPTION
**Pull request by magley**
Closes #4 #7 #8

### CODE REVIEW

File: AlphaVantageGUI/AlphaVantageGUI/Util/ApiUtil.cs [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+        private static string SendRequest(string url)
+        {
+            Uri uri = new(url);
+            using HttpClient client = new();
+
+            Debug.WriteLine($"Sending: {url}");
+
+            HttpRequestMessage request = new(HttpMethod.Get, uri);
+            var response = client.Send(request);
+            return response.Content.ReadAsStringAsync().Result;
```
Comment (filipkontic) [inline AlphaVantageGUI/AlphaVantageGUI/Util/ApiUtil.cs:119]: Can't just do `await client.GetStringAsync(uri)`? See https://learn.microsoft.com/en-us/dotnet/api/system.net.http.httpclient?view=net-7.0#examples commented line

File: AlphaVantageGUI/AlphaVantageGUI/Util/StringUtil.cs [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+            }
+            if (n < 1000000)
+            {
+                return $"{(int)(n / 1000)}K";
+            }
+            if (n < 1000000000)
+            {
+                return $"{(int)(n / 1000000)}M";
+            }
+            return $"{(int)(n / 1000000000)}B";
```
Comment (filipkontic) [inline AlphaVantageGUI/AlphaVantageGUI/Util/StringUtil.cs:27]: Use digit separators

**Review by filipkontic — CHANGES_REQUESTED** [review]

File: AlphaVantageGUI/AlphaVantageGUI/Util/ApiUtil.cs [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+            }
+
+            string url = $"{URL}?function={function}&symbol={symbol}&interval={interval}&market={market}&apikey={Key}";
+            return SendRequest(url);
+        }
+
+        private static string SendRequest(string url)
+        {
+            Uri uri = new(url);
+            using HttpClient client = new();
```
[the remark this answers is not in the document]
Comment (magley) [reply AlphaVantageGUI/AlphaVantageGUI/Util/ApiUtil.cs]: Done in [ca53583](https://github.com/magley/hci-stocks/pull/13/commits/ca53583512142d77fe83e3e0ca1f62ff29f4a07c).

File: AlphaVantageGUI/AlphaVantageGUI/Util/ApiUtil.cs (hunk 1 above)
Comment (magley) [reply AlphaVantageGUI/AlphaVantageGUI/Util/ApiUtil.cs]: Done in [ca53583](https://github.com/magley/hci-stocks/pull/13/commits/ca53583512142d77fe83e3e0ca1f62ff29f4a07c).

File: AlphaVantageGUI/AlphaVantageGUI/Util/StringUtil.cs (hunk 2 above)
Comment (magley) [reply AlphaVantageGUI/AlphaVantageGUI/Util/StringUtil.cs]: Done in [ca53583](https://github.com/magley/hci-stocks/pull/13/commits/ca53583512142d77fe83e3e0ca1f62ff29f4a07c).

**Review by filipkontic — APPROVED** [review]

**Review by swopl — APPROVED** [review]

**Pull request merged**